src/components/timeline/TrackScreenshots.js

```diff
@@ -156,9 +156,7 @@ type HoverPreviewProps = {|
   +trackHeight: number,
 |};
 
-const MAXIMUM_HOVER_SIZE = 350;
-
-const HOVER_MAX_WIDTH_RATIO = 1.75;
+const MAXIMUM_HOVER_SIZE = 300;
```

Comment:
heh, you're still changing this :-)

Reply:
If i change size to 350, 2 tests fails :/
![image](https://user-images.githubusercontent.com/37832457/95775654-a2e62280-0ce4-11eb-8ca4-e78c4f06c97f.png)